File: application/src/main/java/org/curiouscreature/android/roadtrip/MainBox.java
package org.curiouscreature.android.roadtrip;

import android.view.View;

import com.jimulabs.mirrorsandbox.MirrorSandbox;

/**
 * Created by lintonye on 15-02-21.
 */
public class MainBox implements MirrorSandbox {
    private final View mRootView;

    public MainBox(View rootView) {
        mRootView = rootView;
        IntroView introView = (IntroView) mRootView.findViewById(R.id.intro);
        introView.setSvgResource(R.raw.map_usa);
//        introView.stopWaitAnimation();
//        introView.setDrag(0.5f);
    }

    @Override
    public void enterSandbox() {
    }

    @Override
    public void destroySandbox() {

    }
}

File: application/src/main/kotlin/org/curiouscreature/android/roadtrip/MainBox.kt
package org.curiouscreature.android.roadtrip

import android.view.View

import com.jimulabs.mirrorsandbox.MirrorSandbox
import com.jimulabs.mirrorsandbox.MirrorSandboxBase

/**
 * Created by lintonye on 15-02-21.
 */
public class MainBox(private val rootView: View) : MirrorSandboxBase(rootView) {

    override fun `$onLayoutDone`(rootView: View) {
    }

    override fun `$onCreate`(rootView: View?) {
        val introView = rootView?.findViewById(R.id.intro) as IntroView
        introView.setSvgResource(R.raw.map_usa)
    }
}
